lib/create_workout/helper_widgets/submit_button.dart

```diff
@@ -37,32 +37,61 @@ class SubmitButtonState extends State<SubmitButton> {
 
   @override
   Widget build(BuildContext context) {
-    return Container(
-        decoration: BoxDecoration(
-            color: widget.color,
-            // border: Border.fromBorderSide(BorderSide.lerp(a, b, t))
-            borderRadius: const BorderRadius.only(
-                topLeft: Radius.circular(20), topRight: Radius.circular(20))),
-        width: MediaQuery.of(context).size.width,
-        height: MediaQuery.of(context).size.height / 12,
-        child: InkWell(
-          onTap: () {
-            widget.onTap();
-          },
-          child: Ink(
-            height: MediaQuery.of(context).size.height / 12,
-            width: MediaQuery.of(context).size.width,
-            child: Center(
-                child: Padding(
-                    padding: const EdgeInsets.all(10),
-                    child: AutoSizeText(
-                      widget.text,
-                      minFontSize: 18,
-                      // maxFontSize: 200,
-                      style: const TextStyle(
-                          fontWeight: FontWeight.bold, fontSize: 40),
-                    ))),
+    double sizeHeight = MediaQuery.of(context).size.height;
+    double sizeWidth = MediaQuery.of(context).size.width;
+    return Padding(
+      padding: const EdgeInsets.symmetric(horizontal: 20),
+      child: Container(
+          decoration: BoxDecoration(
+              color: widget.color,
+              // border: Border.fromBorderSide(BorderSide.lerp(a, b, t)),
+              borderRadius: BorderRadius.circular((sizeHeight * sizeWidth) * .0005),
           ),
-        ));
+          width: MediaQuery.of(context).size.width,
+          height: MediaQuery.of(context).size.height / 12,
+          child: InkWell(
+            onTap: () {
+              widget.onTap();
+            },
+            child: Ink(
+              height: MediaQuery.of(context).size.height / 12,
+              width: MediaQuery.of(context).size.width,
+              child: SafeArea(
+                child: Stack(
+                  children: [
+                    Center(
+
+                        child: AutoSizeText(
+                          widget.text,
+                          minFontSize: 18,
+                          // maxFontSize: 200,
+                          style: const TextStyle(
+                              fontSize: 25, color: Colors.white),
+                        )
+                    ),
+                    Positioned(
+                      right: 5,
+                      top: 5,
+                      child: Container(
+                        height: MediaQuery.of(context).size.height / 14,
+                        width: MediaQuery.of(context).size.height / 14,
+                        decoration: const BoxDecoration(
+                          color: Color.fromARGB(255,32,132,231),
+                          borderRadius: BorderRadius.all(Radius.circular(35)),
```

Comment:
```suggestion
                          borderRadius: BorderRadius.all(Radius.circular(55)),
```

Thanks for the pull request! One small adjustment from testing on an iPad, please increase the border radius so that it still appears circular on larger screen sizes.

With 35 border radius, appears square on iPad:
![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2024-01-15 at 13 36 17](https://github.com/a-mabe/OpenHIIT/assets/39250511/e0ca403f-6861-457c-bdf8-b092fabf4a6b)

With 55 border radius, appears circular:
![Simulator Screenshot - iPad Pro (12 9-inch) (6th generation) - 2024-01-15 at 13 34 45](https://github.com/a-mabe/OpenHIIT/assets/39250511/d35a4632-446e-47c3-be64-c799f1c64e0a)